Plots/Scatter/NCL_scatter_3.py

```diff
@@ -0,0 +1,97 @@
+"""
+NCL_scatter_3.py
+================
+This script illustrates the following concepts:
+    - Drawing a scatter plot over a map
+    - Choosing marker color, size, and style 
+    - Drawing markers on a map indicating the locations of station data
+   
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/scatter_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/scatter_3_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/scatter_3_2_lg.png
+                         
+"""
+
+###############################################################################
+# Import packages:
+
+import xarray as xr
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from cartopy.mpl.gridliner import LongitudeFormatter, LatitudeFormatter
+import matplotlib.pyplot as plt
+
+from geocat.viz import util as gvutil
+import geocat.datafiles as gdf
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+
+ds = xr.open_dataset(gdf.get("netcdf_files/95031800_sao.cdf"), decode_times=False)
+lat = ds.lat.isel()
+lon = ds.lon.isel()
+
+###############################################################################
+# Plot
+
+def Plots(xlim, ylim, xtic, ytic, xminor, yminor, size, color):
+    '''
+    Creates plot using user specified variables.
+    
+    Parameters
+    ----------
+    xext : 'tuple'
+        Inclusive extent of projection in format (xstart, xend) with values between -180 
+        and 180.
+    yext : 'tuple'
+        Inclusive extent of projection in format (ystart, yend) with values between -90 
+        and 90.
+    xtic : 'int'
+        Step number of x tick label instances in format of number between each tick.
```

Comment:
I would explain that `xtic` and `ytic` are the number of major ticks between -180 and 180. Alternatively, you could specify that this is passed to `range()` as the step argument. I think this would be the clearest explanation for a novice python user.